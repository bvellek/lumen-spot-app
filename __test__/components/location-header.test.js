import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import LocationHeader from '../../public/js/components/location-header';

describe('<LocationHeader />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <LocationHeader />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
