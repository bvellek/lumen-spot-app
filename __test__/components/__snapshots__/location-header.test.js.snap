// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocationHeader /> it renders correctly 1`] = `
<header
  className="search-header"
>
  <div
    className="logo-container"
  >
    <Link
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/"
    >
      <h1>
        <span
          className="visually-hidden"
        >
          Lumen Spot
        </span>
        <SVGLogo />
      </h1>
    </Link>
  </div>
</header>
`;
